src/call/client.rs

```diff
@@ -421,6 +456,14 @@ impl<H: ShareCallHolder, T> ResponseStreamImpl<H, T> {
             }
         }
     }
+
+    // Cancel the call if we still have some messages or did not
+    // receive status code.
+    fn on_drop(&mut self) {
+        if !self.read_done || !self.finished {
```

Comment:
When will `finished` be true while `read_done` be false?

Reply:
When there are some messages buffered in the memory.